spec/smoke_test/job_seekers_can_view_homepage_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'browser_test_helper'
+require 'pry'
+
+RSpec.describe 'Page availability', js: true do
+  context 'Job seeker visits home page' do
+    it 'Page should be available' do
+      page = Capybara::Session.new(:poltergeist)
+      page.visit 'https://teaching-vacancies.service.gov.uk/'
+      expect(page).to have_content('Find a job in teaching')
```

Comment:
With Rails apps that use i18n, it's a good idea to use the i18n helper methods to reference a translation key rather than the translated text, just in case in future the translation changes, so that the tests don't break. So, would recommend using:

```
expect(page).to have_content(I18n.t('jobs.heading')
```

Further reading: https://guides.rubyonrails.org/i18n.html

Reply:
Sure! resolved this in cd60e72